README.md

```diff
@@ -35,6 +35,8 @@ Some projects from CDF already have the Chinese localization documentation. Such
 * [Lan](https://github.com/liangyuanpeng) (Yunhorn)
 * [Shawn](https://github.com/chengjingtao) (Alauda)
 * [Suke](https://github.com/wangsrGit119) (SpringBoot Chinese Community)
+* [Seamus](https://github.com/yJunS) (Jenkins Chinese Community)
+* [Robert Lu](https://github.com/robberphex) (Alibaba Cloud)
```

Comment:
The name list should be in alphabetical order. So, please put Robert before letter S.

`A...R-S...Z`

Reply:
Done.